presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java

```diff
@@ -124,6 +126,9 @@ else if (type.getJavaType() == double.class && columnStatistics.getDoubleStatist
         else if (type.getJavaType() == Slice.class && columnStatistics.getStringStatistics() != null) {
             return createDomain(type, hasNullValue, columnStatistics.getStringStatistics());
         }
+        else if (FLOAT.equals(type) && columnStatistics.getDoubleStatistics() != null) {
+            return createDomain(type, hasNullValue, columnStatistics.getDoubleStatistics(), value -> (long) floatToRawIntBits(value.floatValue()));
```

Comment:
eh, you know it's a double because it's from DoubleStatistics. I think it's safer to keep it as is.